src/Database/LSMTree/Internal/MergeSchedule.hs

```diff
@@ -270,26 +278,50 @@ data Level m h = Level {
 -- | A merging run is either a single run, or some ongoing merge.
 data MergingRun m h =
     -- | A merging of multiple runs.
-    MergingRun !MergePolicyForLevel !NumRuns !(StrictMVar m (MergingRunState m h))
+    MergingRun
+      !MergePolicyForLevel
+      !NumRuns
+      !NumEntries
+      -- ^ Sum of number of entries in the input runs
+      !(UnspentCreditsVar (PrimState m))
+      -- ^ The number of currently /unspent/ credits
```

Comment:
To clarify, these are scaled credits, right?

Reply:
Yes, they are